gravitee-apim-console-webui/src/services/group.service.ts

```diff
@@ -13,9 +13,28 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import * as _ from 'lodash';
 import { PageQuery } from '../entities/pageQuery';
 
+function mapRoles(roles = []) {
+  return Object.entries(roles)
```

Comment:
not sure `roles` has the good type.
Object.entries extracts the key/value of an object, why do it on an array ?